### DISCUSSION TITLE
Google drive is missing?

### DISCUSSION THREAD

**Discussion by ghost** [thebaselab/codeapp#443] [Q&A]
Original Browser 
![image](https://user-images.githubusercontent.com/6639567/167331808-1b3bfb7a-d31b-4e57-9166-7cbdd31882ce.jpeg)

![image](https://user-images.githubusercontent.com/6639567/167331812-1474d277-cfab-42f4-b835-00138a1a8972.jpeg)

![image](https://user-images.githubusercontent.com/6639567/167331842-242c61c2-f9ab-47a2-a90e-99b221769fed.jpeg)

Code

![image](https://user-images.githubusercontent.com/6639567/167331366-28ac8a2a-5c39-4696-b17d-e3e58b5e147a.jpeg)

![image](https://user-images.githubusercontent.com/6639567/167330110-7839baba-5f84-4e8e-b2b4-781736b1a33e.jpeg)

**Comment by HackettLai** [comment]
Oh ya I have the same problem as well. Currently I have to move my project to iCloud

**Reply by luni-moon** [reply]
Please check issues before posting (#152)
